compute_sdk/globus_compute_sdk/serialize/utils.py

```diff
@@ -0,0 +1,32 @@
+def pack_buffers(buffers: list[str]) -> str:
+    """
+    Combines a list of Compute-serialized buffers into a single string format.
```

Comment:
It's already pretty clear in the code, but adding a note here may make it slightly easier?
e.g. "Combines a list of Compute-serialized buffers into a single string format, using chunk length and newline as prefix/delimiter for each chunk"